src/leiningen/new/chestnut/.dir-locals.el

```diff
@@ -1,3 +1,3 @@
 ((nil . ((cider-refresh-before-fn . "reloaded.repl/suspend")
          (cider-refresh-after-fn  . "reloaded.repl/resume")
-         (cider-cljs-lein-repl    . "(do (user/go) (user/cljs-repl))"))))
+         (cider-default-cljs-repl . "(do (user/go) (user/cljs-repl))"))))
```

Comment:
[cider-default-cljs-repl](https://github.com/clojure-emacs/cider/blob/master/doc/clojurescript.md#user-content-starting-a-clojurescript-repl) should reference a REPL type, so this should be more like: 
`(eval . (cider-register-cljs-repl-type 'chestnut "(do (user/go) (user/cljs-repl))"))`
`(cider-default-cljs-repl . chestnut)`